src/jag-mail-it/src/main/resources/application.yml

```diff
@@ -0,0 +1,5 @@
+spring:
+  redis:
+    host: ${REDIS_HOST:localhost}
```

Comment:
can you go with the following pattern: ${SPRING_REDIS_HOST} based on spring.redis.host so we never have to think when we create env variables ?